style.css

```diff
@@ -302,6 +376,17 @@ li:nth-child(1) {
   pointer-events: none;
 }
 
+/*
+.projectPopup {
+  display: flex;
+  flex-direction: column;
+  width: 95vw;
+  height: 90vh;
+  align-items: center;
+  justify-content: center;
+  background-color: #cfd8dc;
+} */
+
```

Comment:
- [ ] To make clean code, kindly delete commented code. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)